recipe/bld.bat

```diff
@@ -15,7 +15,9 @@ cmake -LAH -G"NMake Makefiles"                               ^
     -DCMAKE_BUILD_TYPE=%CMAKE_CONFIG%                        ^
     -DCMAKE_FIND_ROOT_PATH="%LIBRARY_PREFIX%"                ^
     -DCMAKE_INSTALL_PREFIX="%LIBRARY_PREFIX%" ..
-if errorlevel 1 exit 1
+
```

Comment:
Please add:
```
    -DCMAKE_PREFIX_PATH="%PREFIX%"                           ^
    -DCMAKE_CXX_STANDARD:STRING=17                           ^
    -DCMake_HAVE_CXX_MAKE_UNIQUE:INTERNAL=TRUE               ^
```

Reply:
That didn't seem to work on #58

Reply:
It worked, if you see last builds. This failed because of network connection. I will triggered that again

Reply:
https://ci.appveyor.com/project/conda-forge/cmake-feedstock/build/1.0.153